### DISCUSSION TITLE
Support Matrix addresses in Markdown

### DISCUSSION THREAD

**Discussion by notramo** [community/community#9132] [Projects and Issues]
Matrix addresses are incorrectly rendered in Markdown

User addresses are rendered as Github mentions (accidentally mentioning users).
@notramo:grin.hu
Room aliases are not rendered at all.
#cinny:matrix.org

They should be rendered as https://matrix.to links:
[@notramo:grin.hu](https://matrix.to/#/@notramo:grin.hu)
[#cinny:matrix.org](https://matrix.to/#/#cinny:matrix.org)

**Comment by ssbarnea** [comment]
In fact I very interested about posting new discussions topics into matrix rooms. That would heavily boost the usages of discussions on so many open-source projects.

If anyone has such integration done, let me know, maybe we can make it easier to implement.
